### PR TITLE
Added ability to disable TransportActivator, SubscriptionActivator via TransportSettings

### PR DESCRIPTION
**Pull request by kharlamov**

### CODE REVIEW

File: src/FubuTransportation/Subscriptions/SubscriptionActivator.cs [hunk 1]
```diff
@@ -17,18 +16,23 @@ public class SubscriptionActivator : IActivator
...
             _sender = sender;
             _cache = cache;
             _requirements = requirements;
             _graph = graph;
+            _settings = settings;
         }
 
         public void Activate(IEnumerable<IPackageInfo> packages, IPackageLog log)
         {
+            if(_settings.Disabled) return;
```
Comment (RyanHauert) [inline src/FubuTransportation/Subscriptions/SubscriptionActivator.cs]: I don't think you need TransportSettings in this activator if you're handling it in FubuTransportationActivator.

File: ripple.config [hunk 2]
```diff
@@ -38,7 +38,7 @@
     <Dependency Name="FubuPersistence45" Version="0.9.5.95" Mode="Float" />
     <Dependency Name="FubuTestingSupport" Version="1.2.0.259" Mode="Float" />
     <Dependency Name="HtmlTags" Version="2.0.0.181" Mode="Float" />
-    <Dependency Name="LightningQueues" Version="0.1.0.47" Mode="Float" />
+    <Dependency Name="LightningQueues" Version="0.1.0.95" Mode="Fixed" />
```
Comment (RyanHauert) [inline ripple.config]: Is there a reason for Fixing this?

File: src/FubuTransportation/FubuTransportationActivator.cs [hunk 3]
```diff
@@ -14,18 +14,22 @@ public class FubuTransportationActivator : IActivator, IDeactivator
...
             _transports = transports;
             _subscriptions = subscriptions;
             _scheduledJobs = scheduledJobs;
             _pollingJobs = pollingJobs;
+            _settings = settings;
         }
 
         public void Activate(IEnumerable<IPackageInfo> packages, IPackageLog log)
         {
+            if(_settings.Disabled) return;
```
Comment (RyanHauert) [inline src/FubuTransportation/FubuTransportationActivator.cs]: You'll need to do the same in Deactivate()

**Comment by kharlamov** [comment]
MY bad, I believe ripple may have been pulling down an older version.

**Pull request merged**